### DISCUSSION TITLE
Slow start configuration understanding

### DISCUSSION THREAD

**Discussion by anupam-meesho** [projectcontour/contour#6751] [Q&A]
Hi Team,
Some background: We have our whole infra in kubernetes. We have configured contour as an ingress gateway for the means of routing traffic across the clusters.
Question: We have configured slow start for some of our services. We came across two different behaviours for the different pods. One is honouring the slow start other one is takes a way longer time to ramp up the pods to the full capacity. It would be very helpful if you can point to the certain documentation of code that governs this behaviour predictively. Below is the configuration we are using:

```
slowStart:
    aggression: 0.2
    minPercent: 1
    window: 150s
```

<img width="848" alt="slowStart" src="https://github.com/user-attachments/assets/e00e2808-ad22-414d-8fc0-a66299b97303">

**Comment by sunjayBhatia** [comment]
Envoy documentation lives here: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/load_balancing/slow_start
